src/app/shared/services/studio-configuration.ts

```diff
@@ -0,0 +1,14 @@
+interface BlueprintConfiguration {
+  ServerPostrollDuration: number
+}
+
+interface Settings {
```

Comment:
Would be nice if these where called `StudioSettings` so we knew that they were the settings for Studio. Same thing applies for `BlueprintConfiguration` :)